### PR TITLE
Added methods to get timestamp and the password from the passkey.

### PR DESCRIPTION
**Pull request by bdhobare**
Added methods to get timestamp and the password from the passkey instead of hardcoded values.
Also added a sanitizer for phone number. 
_Implemented_ originally [ here.](https://github.com/bdhobare/mpesa-android-sdk)

Added support for Android Studio 3

### CODE REVIEW

File: app/src/main/java/com/myduka/app/activity/MainActivity.java [hunk 1]
```diff
@@ -232,11 +233,11 @@ public void onResponse(okhttp3.Call call, okhttp3.Response response) throws IOEx
 
     public void getPhoneNumber() {
         AlertDialog.Builder builder = new AlertDialog.Builder(this);
-        builder.setTitle("Enter Customer's Safaricom phone number (2547XXX) to checkout Kshs " + String.valueOf(getTotal(prices)));
```
Comment (kisese) [inline app/src/main/java/com/myduka/app/activity/MainActivity.java]: I get the use of number sanitization, just add a comment to make sure developers know that all requests should be in the 254XXX format

File: app/src/main/java/com/myduka/app/activity/MainActivity.java [hunk 2]
```diff
@@ -267,14 +268,15 @@ public void performSTKPush(String phone_number) {
         dialog.setTitle("Please Wait");
         dialog.setIndeterminate(true);
         dialog.show();
+        String timestamp = Utils.getTimestamp();
         STKPush stkPush = new STKPush(Config.BUSINESS_SHORT_CODE,
-                Config.PASSWORD,
-                "20160216165627",
```
Comment (kisese) [inline app/src/main/java/com/myduka/app/activity/MainActivity.java]: This is very helpful, alot of guys have had trouble with this

File: app/src/main/java/com/myduka/app/app/Config.java [hunk 3]
```diff
@@ -26,8 +26,8 @@
 
     //Use credentials from your Lipa na MPESA Online(MPesa Express) App from the developer portal
 
-    public static final String CONSUMER_KEY = "YOUR_CONSUMER_KEY";
-    public static final String CONSUMER_SECRET = "YOUR_CONSUMER_SECRET"; 
+    public static final String CONSUMER_KEY = "YOUR_KEY";
```
Comment (kisese) [inline app/src/main/java/com/myduka/app/app/Config.java]: I'd prefer if these remained as YOUR_CONSUMER_KEY and YOUR_CONSUMER_SECRET for anyone new to Oauth

File: app/src/main/java/com/myduka/app/app/Config.java [hunk 4]
```diff
@@ -44,10 +44,11 @@
 
     //STKPush Properties
     public static final String BUSINESS_SHORT_CODE = "174379";
-    public static final String PASSWORD = "MTc0Mzc5YmZiMjc5ZjlhYTliZGJjZjE1OGU5N2RkNzFhNDY3Y2QyZTBjODkzMDU5YjEwZjc4ZTZiNzJhZGExZWQyYzkxOTIwMTYwMjE2MTY1NjI3";
+    public static final String PASSKEY = "bfb279f9aa9bdbcf158e97dd71a467cd2e0c893059b10f78e6b72ada1ed2c919";
```
Comment (kisese) [inline app/src/main/java/com/myduka/app/app/Config.java]: This is great

**Comment by bdhobare** [comment]
@kisese The suggestions have been applied.Thanks

**Pull request merged**